pyartcd/pyartcd/github.py

```diff
@@ -0,0 +1,166 @@
+import requests
+import base64
+import os
+import ruamel.yaml
+from pyartcd import exceptions
+
+
+class GithubClient:
```

Comment:
I rewrote the code to use [ghapi](https://github.com/fastai/ghapi) as it covers the entire entire [GitHub API](https://docs.github.com/rest) but I think it supports only from Python 3.7 (since I'm getting a `SyntaxError: future feature annotations is not defined` error which exists in 3.6, when I try to run the build in my hack space). Which version is our Jenkins on?

Reply:
I tried using ghapi, but its not supported by the Python 3.6, which is what were on. I also looked into PyGithub, but I need to use the function [merge-upstream](https://docs.github.com/en/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository) which hasn't been added to PyGithub yet...

Reply:
Used ghapi and added extra functions for the `merge-upstream` feature.

Reply:
Done!